content/en/docs/setup/install/helm/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Customizable Install with Helm
+description: Install and configure Istio for in-depth evaluation or production use.
+weight: 27
+keywords: [kubernetes,helm]
+owner: istio/wg-environments-maintainers
+icon: helm
+test: no
+---
+
+Follow this guide to install and configure an Istio mesh using
+[Helm](https://helm.sh/docs/) for in-depth evaluation or production use.
+
+The Helm charts used in this guide are the same underlying charts used when
+installing Istio via [Istioctl](/docs/setup/install/istioctl/) or the
+[IstioOperator API](/docs/setup/install/operator/).
+
```

Comment:
I'd mention the following caveats:
- schema is not checked 
- checks for deprecated values are not done
- values.yaml API is likely to change to align with IstioOperator API
- patching the output manifest (https://istio.io/latest/docs/setup/install/istioctl/#patching-the-output-manifest)  requires kustomize.
- istioctl upgrade command cannot be used with Helm installations

Reply:
I am hesitating to add them as all the other supported install methods have equal downsides that we don't explain.

1. "istioctl" install is an imperative command that mixes declarative intent with custom logic built from Istio community which changes every version. Things that can happen without users noticing if you change your K8s cluster silently include toggling between 1p/3p JWT and it's associated ramifications on new pods vs existing workloads. No longer get various default Ingress specific rules, a new install will not have Telemetry addons with latest istioctl but old installs will have all addons
2. Operator can be an escape hatch for CVEs due to broad permissions and possibly less battle tested/mature version of Helm 2 Tiller running in your cluster. Most of the problems I said above happen with Operator too as between versions a lot can be hidden and surprising for users.

Happy to add caveats but we should be clear about pitfalls of all other options so users can choose the best tool based on their needs.

Reply:
I believe speaking poorly about our own multi-path installation tooling won't help us. Even mentioning the tradeoffs, as both @ostromart and @nrjpoddar have done a great job on, can be misconstrued by those outside the community to indicate we think Helm is evil or we think Istioctl is evil or the standalone operator is evil. We have three paths for a reason. 👍 

It is better to keep things positive if there are any comparisons to make.

One thing I think is important to point out, however, is that `istioctl` is stable and tested in CI - and these are comparisons we should be free to make. Our Helm installation - using Helm-based workflows is experimental or alpha and lacks CI. I think it's fair to reference the feature status pages for [istiocl](https://github.com/istio/istio.io/blob/master/content/en/about/feature-stages/index.md#core) and add an alpha feature for Helm based installation as part of a new feature in the [enhancements](https://github.com/istio/enhancements/issues/new?template=alpha_feature.md) repository.

Until the new Helm charts migrate through the various feature stages, I think we will have a hard time claiming the Helm charts as they stand are for production usage in Helm workflows, as we lack any CI for Helm workflows with Istio's charts.

Reply:
the intent there is not to bad mouth helm but rather point out features to help users make a choice between the installation options. we do mention the security implications of operator and we should certainly mention any "headline" downsides for istioctl - the items there a bit too detailed to be appropriate imo but perhaps they can be rephrased to be more suitable. anyway, i'm fine with putting all this into a section that helps users choose an option.

Reply:
@ostromart & @sdake  I agree that we should document tradeoffs of all available mechanisms so users can make an informed decision.

Let's work together to find the correct way to add the helpful content we all want to highlight and the appropriate language, etc.

I do think that we **MUST** highlight the maturity of Helm based install and I will add that a follow-on.